src/components/Validate/index.tsx

```diff
@@ -94,14 +99,16 @@ const Validate = ({ toggleSnackbar, setSnackbarMessage }: IHomeProps) => {
       aspect: '' as string,
       howKnown: '' as string,
       effectiveDate: new Date(),
-      stars: null as number | null
+      stars: null as number | null,
+      amt: null as number | null,
+      source: '' as string
```

Comment:
I think it will not work because the end points of backend is not ready to revieve this values and this values not expected